spec/support/features/webauthn_helper.rb

```diff
@@ -11,6 +11,14 @@ def mock_webauthn_setup_challenge
     )
   end
 
+  def mock_setup_eligible_user_device
```

Comment:
- Should we just update the [default user agent string for mobile](https://github.com/18F/identity-idp/blob/c459137c5d4ac4cd28794c7e891379d3c386a8a7/spec/support/capybara.rb#L24-L26) to be able to avoid this?
- I'd suggest incorporating "webauthn" somewhere in the name of the method, even if it's in `WebauthnHelper`, to clarify at call-sites, and for consistency with other methods in this file.